docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -218,134 +203,139 @@ This is handy because it means you can have the bulk of your code in `spare.xml`
 
-```
+```xml
 <view name="system">
 
 </view>
 ```
 
-Now we add some **Elements** to the View. To start with we'll change the background color by creating a new Image Element called 'background_color'.
+Now we add some **Elements** to the View. To start with we'll change the background color by creating a new Image Element called `background_color`.
 
-```
+```xml
 <view name="system">
     <image name="background_color" extra="true"></image>
 </view>
 ```
 
->The `extra="true"` means that this is a new **Element** being added, not something that is already defined by Emulationstation
+**The `extra` attribute**
+
+> The `extra="true"` XML attribute means that this is a new XML **Element** being added, not something that is already defined by EmulationStation
+
+We need to add some **Properties** (as XML child elements) to the Element to tell it what to do. There are seven Properties accepted by the image Element. They are:
 
-We need to add some **Properties** to the Element to tell it what to do. There are 7 Properties accepted by the Image Element. They are:
+- `<pos>`: The position of the image within the screen, given in percentage of the actual display resolution.
+- `<size>`:The absolute size of the image given in percentage of the actual display resolution. Does not maintain aspect ratio. Use either this or `<maxSize>`, not both.
+- `<maxSize>`: The maximum size to stretch an image. Maintains aspect ratio. Use either this or `<size>`, not both.
+- `<origin>`: The origin point of the image. Default is top-left.
+- `<path>`: The relative path to the image file.
+- `<tile>`: If the image repeats or not.
+- `<color>`: A color overlay, allowing you to change the color and opacity of the image. Uses four bytes values. The first three bytes represent the Hex color code (one byte per red, green, blue) with the last byte is defining the opacity (alpha channel).
 
-- `<pos>`
-  - The position of the image within the screen.
-- `<size>`
-  - The absolute size of the image. Does not maintain aspect ratio. Use either this or `<maxSize>`, not both.
-- `<maxSize>`
-  - The maximum size to stretch an image. Maintains aspect ratio. Use either this or `<size>`, not both.
-- `<origin>`
-  - The origin point of the image. Default is top-left.
-- `<path>`
-  - The relative path to the image file.
-- `<tile>`
-  - If the image repeats or not.
-- `<color>`
-  - A color overlay, allowing you to change the color and opacity of the image.
-  - Uses 8 digit Hex values. The first 6 digits represent the Hex color code with the last 2 digits controlling the opacity.
+If you want a more detailed explanation of Element Properties you can read [this section](https://github.com/Aloshi/EmulationStation/blob/master/THEMES.md#image) of the EmulationStation themes documentation. Especially if you want to use absolute pixel sizes (see [`<resolution/>` element](https://github.com/RetroPie/EmulationStation/blob/master/THEMES.md#the-resolution-tag))
```

Comment:
1st link to the themes doc should be also to RetroPie's repository.

Reply:
Fixed.